android/src/main/java/com/mobilepaymentssdkreactnative/MobilePaymentsSdkReactNativeModule.kt

```diff
@@ -258,6 +262,104 @@ class MobilePaymentsSdkReactNativeModule(private val reactContext: ReactApplicat
     }
   }
 
+  @ReactMethod
+  fun getReaders(promise: Promise) {
+    val readerManager = MobilePaymentsSdk.readerManager()
+    val readers = readerManager.getReaders()
+    val readerList = Arguments.createArray()
+    readers.forEach {
+      readerList.pushMap(it.toReaderInfoMap())
+    }
+    promise.resolve(readerList)
+  }
+
+  @ReactMethod
+  fun getReader(id: String, promise: Promise) {
+    val readerManager = MobilePaymentsSdk.readerManager()
+    val reader = readerManager.getReader(id)
+    val readerMap = reader?.toReaderInfoMap()
+    promise.resolve(readerMap)
+  }
+
+  @ReactMethod
+  fun forget(id: String, promise: Promise) {
+    val readerManager = MobilePaymentsSdk.readerManager()
+    val reader = readerManager.getReader(id)
```

Comment:
Here and elsewhere, I assume you have the extra checks because you don't have an "ApiReader" type?